src/plugins/direct_access_links/public/plugin.ts

```diff
@@ -0,0 +1,85 @@
+/*
```

Comment:
Would be nice to add some Jest tests for this file.

Reply:
yesss, def, thanks for the reminder!

Reply:
<img width="1052" alt="Screen Shot 2020-02-21 at 12 26 17 PM" src="https://user-images.githubusercontent.com/16563603/75056647-75bed880-54a5-11ea-9f3e-45ff24770a8e.png">